src/routes/results/[results_string]/+page.svelte

```diff
@@ -28,48 +29,58 @@
 	});
 </script>
 
-<main>
-	<h1 class="title">Your Results</h1>
-	<div class="chart" aria-hidden="true" bind:clientWidth={chartWidth}>
-		<SpiderChart answers={data.object} {highlight} {chartWidth} />
+<div class="outer">
+	<div class="logo">
+		<Logo />
 	</div>
-	<div class="results">
-		<ol>
-			{#each dynamics as dynamic, idx}
-				<li class:highlight={idx === highlight}>
-					{dynamic.full}
-					<span class="visually-hidden">Your answer: {data.answers[idx].value} out of 5</span>
-				</li>
-			{/each}
-		</ol>
-		<div class="next-steps">
-			<h2>If you have a notebook on hand, jot down your responses.</h2>
-			<ul>
-				<li>Looking at your results, what insights arise for you?</li>
-				<li>
-					Given what feels wobbly and what feels strong, pick one dynamic that might support your
-					own deep, sustained, and courageous climate engagement. How could you invest in it with
-					intention? What support do you need?
-				</li>
-			</ul>
-			<div class="actions">
-				<!-- TODO: add actions and link -->
-				<button class="btn secondary">Email Your Results</button>
-				<a href="#" class="btn secondary">Resources for Support</a>
+	<main>
+		<h1 class="title">Your Results</h1>
+		<div class="chart" aria-hidden="true" bind:clientWidth={chartWidth}>
+			<SpiderChart answers={data.object} {highlight} {chartWidth} />
+		</div>
+		<div class="results">
+			<ol>
+				{#each dynamics as dynamic, idx}
+					<li class:highlight={idx === highlight}>
+						{dynamic.full}
+						<span class="visually-hidden">Your answer: {data.answers[idx].value} out of 5</span>
+					</li>
+				{/each}
+			</ol>
+			<div class="next-steps">
+				<h2>Take a moment to pause and reflect.</h2>
+				<p>If you have a notebook on hand, jot down your responses.</p>
+				<ul>
+					<li>Looking at your results, what insights arise for you?</li>
+					<li>
+						Given what feels wobbly and what feels strong, pick one dynamic that might support your
+						own deep, sustained, and courageous climate engagement. How could you invest in it with
+						intention? What support do you need?
+					</li>
+				</ul>
+				<div class="actions">
+					<!-- TODO: add email action -->
+					<button class="btn secondary">Email Your Results</button>
+					<a href="https://www.allwecansave.earth/dynamics-resources" class="btn secondary"
+						>Resources for Support</a
+					>
+				</div>
 			</div>
 		</div>
-	</div>
-</main>
+	</main>
+	<footer>
+		© 2024 The All We Can Save Project. Developers <a
+			href="https://github.com/chelshaw"
+			target="_blank"
+			rel="noopener nofollow">Chelsea Shaw</a
+		>, Emi Jackson, and Helen Li brought this interactive tool to life.
```

Comment:
ooh nice, could you add my GH link too? `https://github.com/emikjackson` 🙏

Reply:
perhaps @li-helen wants hers added too?

Reply:
ah I had missed your note on this on slack, thanks chelsea!

Reply:
Went ahead and added both, we can always remove if @li-helen prefers a different link 😄